velox/expression/tests/ExpressionFuzzer.cpp

```diff
@@ -426,12 +437,24 @@ core::TypedExprPtr ExpressionFuzzer::generateArgConstant(const TypePtr& arg) {
       vectorFuzzer_.fuzzConstant(arg, 1));
 }
 
+// Either generates a new column of the required type or if already generated
+// columns of the same type exist then there is a 30% chance that it will
+// re-use one of them.
 core::TypedExprPtr ExpressionFuzzer::generateArgColumn(const TypePtr& arg) {
-  inputRowTypes_.emplace_back(arg);
-  inputRowNames_.emplace_back(fmt::format("c{}", inputRowTypes_.size() - 1));
-
+  auto& listOfCandidateCols = typeToColumnNames_[arg->toString()];
+  bool addNewCol = !FLAGS_velox_fuzzer_enable_column_reuse ||
+      listOfCandidateCols.empty() || !vectorFuzzer_.coinToss(0.3);
```

Comment:
nit: The other way around might be more straightforward, i.e., `bool reuseCol = FLAGS_velox_fuzzer_enable_column_reuse && listOfCandidateCols.size() && vectorFuzzer_.coinToss(0.3)`. Ditto for generateExpression(). But not a strong opinion.